helpers/3p/utils/lib/markdown.js

```diff
@@ -1,41 +1,53 @@
 const merge = require('./mixinDeep');
-const { Remarkable } = require('remarkable');
+const {Remarkable} = require('remarkable');
+const {linkify} = require('remarkable/linkify');
 
 module.exports = function markdown(config) {
 
-  if (typeof config === 'string') {
-    return helper.apply(this, arguments);
-  }
-
-  config = config || {};
-  if (config.fn || config.hash || arguments.length > 1) {
-    console.log(arguments);
-    return helper.apply(this, arguments);
-  }
-
-  function helper(context, options) {
-    if (typeof context === 'string') {
-      var opts = merge({}, config, options);
-      console.log(options)
-      var md = new Remarkable(opts);
-      return md.render(context);
+    if (typeof config === 'string') {
+        return helper.apply(this, arguments);
     }
 
-    if (typeof context === 'object' && typeof context.fn === 'function') {
-      options = context;
-      context = {};
+    config = config || {};
+    if (config.fn || config.hash || arguments.length > 1) {
+        return helper.apply(this, arguments);
     }
 
-    options = merge({ html: true, breaks: true }, config, options);
-    options = merge({}, options, options.markdown, options.hash);
-    if (options.hasOwnProperty('lang')) {
-      options.langPrefix = options.lang;
+    function helper(context, options) {
+
+        if (typeof context === 'string') {
+            const opts = merge({}, config, options);
+            const md = buildRemarkable(opts);
+            return md.render(context);
+        }
+
+        if (typeof context === 'object' && typeof context.fn === 'function') {
+            options = context;
+            context = {};
+        }
+
+        options = merge({html: true, breaks: true}, config, options);
+        options = merge({}, options, options.markdown, options.hash);
+        if (options.hasOwnProperty('lang')) {
+            options.langPrefix = options.lang;
+        }
+
+        const md = buildRemarkable(options);
+        const ctx = merge({}, options, (this.context || this), context);
+        return md.render(options.fn(ctx));
     }
 
-    var md = new Remarkable(options);
-    var ctx = merge({}, options, (this.context || this), context);
-    return md.render(options.fn(ctx));
-  }
+    function buildRemarkable(options) {
+        let remarkable;
+        if (options.linkify === true) {
+            remarkable = new Remarkable(options).use(linkify);
+        } else {
+            remarkable = new Remarkable(options);
+        }
```

Comment:
```suggestion
        const remarkable = options.linkify === true
            ? new Remarkable(options).use(linkify)
            : new Remarkable(options);
```

This might look cleaner if we use ternary operators here, and then we don't need a mutable variable

Reply:
done